.travis/install.sh

```diff
@@ -4,8 +4,6 @@ set -e
 set -x
 
 if [[ "$(uname -s)" == 'Darwin' ]]; then
-    brew update || brew update
```

Comment:
Its not, but I finally snapped and removed it after it failed yet another couple of my macos builds.